roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
@erikh @mapuri: Latest contivctl net command seems to be broken/missing the global level options. ACI mode cannot be set as a result of that. I will raise an issue in contivctl repository for this. Until that is fixed, this seems to be the only way to proceed. I can merge in after the change goes in if required. But I'll leave it upto @jainvipin to take the call for the priority on this.